events/hfe_events_init.txt

```diff
@@ -845,26 +849,33 @@ country_event = {
 			}
 			random_system_planet = {
 				limit = {
-					or = {
+					OR = {
 						has_planet_flag = planet_earth
 						has_planet_flag = nuked_earth
-						and = {
+						AND = {
 							exists = owner
 							is_capital = yes
 						}
 					}
 				}
-				#log = "hfe find earth"
 				save_global_event_target_as = hfe_earth
+				if = {
+					limit = { has_global_flag = HFE_TSP_spawn_test }
+					log = "hfe find earth"
+				}
 			}
 		}
 		# create need species
 		# egypt
 		create_species = {
 			name = "Ju Ahet"
-			plural = "Ju Ahet"
+			plural = "Ju Ahets"
+			adjective = "Ju Ahetese"
 			class = HUM
 			portrait = hfe_portrait_egypt
+			name_list = cybrxkhan_fantasy_egyptian
+			extra_trait_points = 10
+			can_be_modified = no
```

Comment:
I probably will change it in future

Reply:
The reason I made the Ju Ahets unable to be modified, though, is because a player that is playing as a human with even the most basic genetic technology can turn themselves into Ju Ahets. They are crazy OP in the early game, and those Extra Trait Points would make them invincible.